transformations/slangificator/README.md

```diff
@@ -0,0 +1,35 @@
+# Slangificator 🦎  + ⌨️ → 🐍
+This transformation replaces some of the words (in particular, nouns, adjectives, and adverbs) of the original text with their corresponding slang. The replacement is done with the subset of the "Dictionary of English Slang & Colloquialisms". The amount of replacement is proportional to the corresponding probabilities of replacement.
+
+Author name: Denis Kleyko
+
+Author email: denis.kleyko@gmail.com
+
+Author Affiliation: University of California, Berkeley and Research Institutes of Sweden
+
+## What type of a transformation is this?
+This transformation is intended to model slang by taking the original text and randomly replacing those words in the text, which have their slang correspondence. 
+
+## What tasks does it intend to benefit?
+This perturbation would benefit all tasks which have a sentence/paragraph/document as input like text classification, 
+text generation, etc. 
+
+## Robustness Evaluation
+The outcome of running 'python evaluate.py -t Slangificator' was as follows:
+
+The accuracy on this subset which has 1000 examples = 96.0
+
+Finished transformation! 1000 examples generated from 1000 original examples, with 999 successfully transformed and 1 unchanged (0.999 perturb rate)
+
+Here is the performance of the model on the transformed set
+
+The accuracy on this subset which has 1000 examples = 93.0
```

Comment:
It would be great if a [robustness evaluation](https://github.com/GEM-benchmark/NL-Augmenter/tree/main/evaluation) is conducted to be added to the leaderboard.

Reply:
I only managed to run "evaluate.py" for "textattack/roberta-base-imdb" getting some Torch CUDA-related issues for the other three models.

Reply:
Okay, I can take a look once all PRs are merged but if you get a chance, please create a separate PR for the robustness evaluation.